src/indexer/index_writer.rs

```diff
@@ -654,11 +657,27 @@ mod tests {
         let index = Index::create_in_ram(schema_builder.build());
         let _index_writer = index.writer(40_000_000).unwrap();
         match index.writer(40_000_000) {
-            Err(Error(ErrorKind::FileAlreadyExists(_), _)) => {}
+            Err(TantivyError::LockFileAlreadyExists(_)) => {}
             _ => panic!("Expected FileAlreadyExists error"),
         }
     }
 
+    #[test]
+    fn test_lockfile_already_exists_error_msg() {
+        let schema_builder = schema::SchemaBuilder::default();
+        let index = Index::create_in_ram(schema_builder.build());
+        let _index_writer = index.writer(40_000_000).unwrap();
```

Comment:
use one thread and a `3MB` buffer

Reply:
@petr-tik up

Reply:
Paul, thanks for following up. I have been travelling and jet lagged, should get some time tonight